rero_ils/modules/acq_order_lines/dumpers.py

```diff
@@ -66,3 +67,29 @@ def dump(self, record, data):
         }
         data['document'] = {k: v for k, v in data['document'].items() if v}
         return data
+
+
+class AcqOrderLineNotificationDumper(InvenioRecordsDumper):
+    """Order line dumper class for acquisition order."""
+
+    def dump(self, record, data):
+        """Dump an AcqOrderLine instance for acquisition order notification.
+
+        :param record: The record to dump.
+        :param data: The initial dump data passed in by ``record.dumps()``.
+        """
+        # Dumps AcqOrderLine acquisition
+        data.update({
+            'quantity': record.get('quantity'),
+            'note': record.get_note(AcqOrderLineNoteType.VENDOR)
+        })
+        data = {k: v for k, v in data.items() if v}
+
+        # Add document informations: formatted title and ISBN identifiers.
+        document = record.document
+        data['document'] = {
+            'title': title_format_text_head(document.get('title', [])),
+            'identifiers': document.get_identifier_values(filters=['bf:Isbn'])
+        }
+        data['document'] = {k: v for k, v in data['document'].items() if v}
```

Comment:
Do we use this a lot maybe a utils function could be helpful:
` {k: v for k, v in data['document'].items() if v}`